@xen-orchestra/web-core/lib/components/info/UiInfo.vue

```diff
@@ -0,0 +1,56 @@
+<!-- v2 -->
+<template>
+  <div class="vts-ui-info">
```

Comment:
To match the new name:
```suggestion
  <div class="vts-info">
```